### DISCUSSION TITLE
Allow multiple authors, illustrators, editors, etc.

### DISCUSSION THREAD

**Discussion by willsheppard** [mateusz-bak/openreads#707] [Ideas]
When I add a book, even when it's recognised, only one author name appears. I want to log all the authors, especially with children's picture books that always have an illustrator.

Would you consider adding any number of author/illustrator/editor fields?

**Describe alternatives you've considered**

Comma separated names in the author field. Not really great.

**Comment by willsheppard** [comment]
Actually I'm a little bit ignorant of how the data is stored. Does OpenReads have its own public database of books? How does this compare to other public book databases?

Are there a standard set of fields for other book databases that could be used here?

**Comment by mateusz-bak** [comment]
Hi, Openreads uses OpenLibrary as its database. 

Can you provide examples of books that are missing multiple authors but they are present on the record in https://openlibrary.org ?

**Comment by exstntlmsnthrp** [comment]
@willsheppard and @mateusz-bak  Unfortunately, one of the few weaknesses of openlibrary is they often omit or limit the listing of multiple authors or contributors. Many of the books in my collection have multiple authors and thus far, only one author ever shows up in openlibrary. 

It would be a brilliant feature if a user could add additional fields in openreads, such as additional authors or illustrators or other contributors. To this point, I've been adding additional authors on the same line, but it's inconvenient.

**Comment by lucastucious** [comment]
> Hi, Openreads uses OpenLibrary as its database. 
> 
> Can you provide examples of books that are missing multiple authors but they are present on the record in https://openlibrary.org ?

https://openlibrary.org/works/OL24153159W/D_D_Player%27s_Handbook?edition=key%3A/books/OL49829871M

Here is a book with multiple authors
